README.md

```diff
@@ -3,132 +3,339 @@
+- A proper C/C++ compiler toolchain for the given platform
+    - Windows:
+        - [Visual C++ Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/) or a recent version of Visual C++ (e.g. [the free Community](https://www.visualstudio.com/products/visual-studio-community-vs))
+    - Unix/Posix:
+        - Clang or GCC
+
+#### Notes
+- Additional dependencies [Open MCT](https://github.com/nasa/openmct) (Open source visualization framework by NASA) and [YarpJS](https://github.com/robotology/yarp.js) (Yarp Javascript bindings) are automatially installed by the Node.js `npm` package manager when installing the repository main modules (steps 7 and 8).
+- [YARP](https://github.com/robotology/yarp), [CMake](http://www.cmake.org/download/) and C/C++ compiler toolchain are transitory dependencies which shouldn't be mentioned a priori here, but since they are not handled by the [YarpJS](https://github.com/robotology/yarp.js) installation process (via the package manager `npm`), they need to be manually installed.
 
-Client dependencies:
+### Client dependencies
 - Browser: [Google Chrome](https://www.google.com/chrome), [Mozilla Firefox](https://www.mozilla.org/en-US/firefox/products), Apple Safari, etc.
 
+### Simulation Dependencies
+- [Gazebo](https://gazebosim.org): the simulator (tested Gazebo 11).
+- [YARP](https://github.com/robotology/yarp): Middleware for handling the communication with the robot.
+- [icub-models](https://github.com/robotology/icub-models): provide the iCub models.
+- [GazeboYARPPlugins](https://github.com/robotology/gazebo-yarp-plugins.git): Plugins interfacing the iCub simulation model with YARP device wrappers.
 
-## [Server Installation](#top)
 
-The following instructions assume you are installing the software as a non-root user. Make sure that you have [Git](https://git-scm.com) and the . The installation and run has been tested on MacOS Catalina 10.15.7.
+The following instructions assume you are installing the software as a non-root user. Make sure that you have [Git](https://git-scm.com) installed in your platform. The tool installation and execution have been tested on MacOS Catalina 10.15.7 and Vanilla Ubuntu 20.04.
+
+## Installation Prerequisites For Connecting to a Local Robot Network
+
+The scope of this installation guide addresses only the modules required for running the **Yarp-OpenMCT** tool. You won't need additional modules if you are connecting your machine to a local robot network which has the following modules already installed and running:
+- a `yarprobotinterface` on the iCub head,
+- a Yarp Name Server on the robot network,
+- the `wholeBodyDynamics` device from `whole-body-estimators` repository, for running the FT sensors calibration,
+- the `WalkingModule` module from the `walking-controllers` repository, for running the walking controller,
+- the `icubBattery` or the `fakeBattery` device and wrapper for the battery status.
+- the camera driver and wrapper for the iCub cameras.
```

Comment:
I would suggest mentioning that these are not hard requirements anyway

Reply:
Good point!

Reply:
I actually refactored this part, reducing it to the minimum. I moved the list of ports to the "how to run" section (list depending on which data the user wishes to visualize).